about.html

```diff
@@ -0,0 +1,223 @@
+                          <img src="./img/logo.png" alt="logo" />
+                        </a>
+                      </li>
+                      <li>
+                        <ul class="nav-wrap">
+                          <li>
+                            <a href="about.html" class="nav-links">About</a>
+                          </li>
+                          <li><a href="#" class="nav-links">Program</a></li>
+                          <li>
+                            <a href="#works" class="nav-links">Buy Tickets</a>
+                          </li>
+                          <li>
+                            <a href="#works" class="nav-links">Sponsor</a>
+                          </li>
+                          <li><a href="#works" class="nav-links">News</a></li>
+                          <li class="cc">
+                            <a href="index.html" class="textcc">
+                              Fundraising Concert
+                            </a>
+                          </li>
+                        </ul>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+              </div>
+            </nav>
+          </header>
+          <div class="intro about-wrap">
+            <p id="introText" class="hide">"Hello! Sharing World"</p>
+            <h1 class="heading mb-4 me-3">
+              Fund Raising
+              <br />
+              Concert in Myanmar 2023
+            </h1>
+          </div>
+          <div class="mb-lg-5 detail-wrap about-wrap">
+            <p class="detail detail-about">
+              The concert brings together the community of activists, experts
+              and all the people around the world. This year, we hope to expand
+              our invitation list including organizations and individuals
+            </p>
+            <p class="contact contact-about">
+              Please contact us per Email for any further questions about the
+              Fund Raising concert 2023!
+              <br />
+              <a href="#" class="fundLink">fundraising2023@icmyanmar.org</a>
+            </p>
+          </div>
+        </section>
+        <section class="mb-5">
+          <div class="p-3 text-center concert">
+            <div class="p-4">
+              <h2>
+                Iron Cross Fund Raising Concert 2023 Logo
+              </h2>
+              <div class="d-flex align-items-center justify-content-center">
+                <hr class="line" />
+              </div>
+              <p>
+                The logo of Iron Cross Fund Raising concert 2023 was decided
+                through the logo competition from 8.June to 7.July.
+              </p>
+            </div>
+            <div class="cross-wrap">
+              <div class="border mt-2 iron-cross">
+                <img src="./img/logo.png" alt="Iron Cross Logo" />
+              </div>
+            </div>
+          </div>
```

Comment:
- [ ] The `Iron Cross Fund Raising Concert` section should be a section and not a `div` inside a section. this is the proper page structure, kindly fix it